addons/addon-base/packages/services/lib/user/__tests__/user-authz-service.test.js

```diff
@@ -0,0 +1,68 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+const ServicesContainer = require('@aws-ee/base-services-container/lib/services-container');
+const UserAuthzService = require('../user-authz-service');
+const SystemContext = require('../../helpers/system-context');
+
+describe('UserAuthzService', () => {
+  let service = null;
+  beforeEach(async () => {
+    // Initialize services container and register dependencies
+    const container = new ServicesContainer();
+
+    container.register('userAuthzService', new UserAuthzService());
+    await container.initServices();
+
+    // Get instance of the service we are testing
+    service = await container.find('userAuthzService');
+  });
+
+  describe('authorizeCreate test', () => {
+    it('should allow system to create root user', async () => {
+      // BUILD
+      const sysContext = SystemContext.getSystemRequestContext();
```

Comment:
Please see my comment above regarding using camel case for systemContext instead of pascal case.